app/scripts/directives/deployImage.js

```diff
@@ -387,7 +394,10 @@ angular.module("openshiftConsole")
           // button is outside the component since it is in the wizard footer. Listen
           // for an event for when the button is clicked.
           $scope.$on('newAppFromDeployImage', $scope.create);
-          $scope.$on('$destroy', hideErrorNotifications);
+          $scope.$on('$destroy', function() {
+            hideErrorNotifications();
+            noProjectsCantCreateWatch();
```

Comment:
@dtaylor113 This is fine, but not needed. The $scope listeners are already deregistered on destroy. I'd honestly rather remove this logic from the controllers to keep the code simpler. It can just be...

```js
$scope.$on('no-projects-cannot-create', function() {
  $scope.noProjectsCantCreate = true;
});
```

in each of the controllers.

Reply:
Ok, fixed